block/manager.go

```diff
@@ -347,8 +356,36 @@ func (m *Manager) IsDAIncluded(hash types.Hash) bool {
 	return m.blockCache.isDAIncluded(hash.String())
 }
 
+// getRemainingSleep calculates the remaining sleep time based on config and a start time.
+func (m *Manager) getRemainingSleep(start time.Time) time.Duration {
+	elapsed := time.Since(start)
+	interval := m.conf.BlockTime
+
+	if m.conf.LazyAggregator {
+		if m.buildingBlock {
+			interval = m.conf.BlockTime
```

Comment:
isn't this line redundant given the interval is set to `m.conf.BlockTime` above?

Reply:
yeah, it's redundant now